index.js

```diff
@@ -10,7 +10,7 @@ try {
   const product = input1 * input2;
 
   // Create a custom message
-  const message = `The inputs are ${input1} and ${input2}. Sum is ${sum}, product is ${product}.`;
```

Comment:
The variable `foobar` is used instead of `input1` in the message string. This appears to be a typo or unintended change that will cause incorrect output. Please double check and use the correct variable name.